words/objects.txt

```diff
@@ -222,21 +251,23 @@ book
 bookcase
 booklet
 boot
+booth
 border
 botany
 bottle
 bottom
 bovid
 bow
 bowl
+box
```

Comment:
Occasionally used as a euphemism, but probably generic enough to be fine

Reply:
I agree that its pretty generic. we can always remove it in the future if needed.